README.md

```diff
@@ -54,7 +54,7 @@ Also check [Before you install](https://docs.bigbluebutton.org/2.4/install.html#
 | | `bbb_turn_enable` | enable the use uf TURN in general | `yes` | |
 | | `bbb_stun_servers` | a list of STUN-Server to use | `{{ bbb_hostname }}` | an array with key `server` - take a look in defaults/main.yml |
 | | `bbb_ice_servers` | a list of RemoteIceCandidate for STUN | `[]` | in array with key `server` |
-| | `bbb_turn_servers` | a list of TURN-Server to use | `{}` | take a look in defaults/main.yml |
```

Comment:
Why did this doc line get deleted?
Its getting used:
https://github.com/ebbba-org/ansible-role-bigbluebutton/blob/85eb1cce7a95711a49d8489fe6283ad5524106e5/templates/bbb/turn-stun-servers.xml.j2#L43-L50

https://github.com/ebbba-org/ansible-role-bigbluebutton/blob/85eb1cce7a95711a49d8489fe6283ad5524106e5/defaults/main.yml#L18-L28